docs/source/challenge-doc.rst

```diff
@@ -60,21 +68,23 @@ Observation Space
 +-----------------------------------------+-----------------------------+-----------------+
 | Joint positions of myoArm               | obs_dict['myohand_qpos']    | (38x1)          | 
 +-----------------------------------------+-----------------------------+-----------------+
-| Joint velocity of myoArm                | obs_dict['myohand_qvel"]    | (38x1)          |
+| Joint velocity of myoArm                | obs_dict['myohand_qvel']    | (38x1)          |
++-----------------------------------------+-----------------------------+-----------------+
+| Joint positions of MPL                  | obs_dict['pros_hand_qpos']  | (27x1)          |
 +-----------------------------------------+-----------------------------+-----------------+
-| Joint positions of MPL                  | obs_dict['pros_hand_qpos"]  | (27x1)          |
+| Joint velocity of MPL                   | obs_dict['pros_hand_qvel']  | (26x1)          |
 +-----------------------------------------+-----------------------------+-----------------+
-| Joint velocity of MPL                   | obs_dict['pros_hand_qvel"]  | (27x1)          |
+| Joint positions of object               | obs_dict['object_qpos']     | (7x1)           |
 +-----------------------------------------+-----------------------------+-----------------+
-| Joint positions of object               | obs_dict['object_qpos"]     | (7x1)           |
+| Joint velocity of object                | obs_dict['object_qvel']     | (6x1)           |
 +-----------------------------------------+-----------------------------+-----------------+
-| Joint velocity of object                | obs_dict['object_qvel"]     | (6x1)           |
+| Touching information of object          | obs_dict['touching_body']   | (5x1)           |
```

Comment:
replace with - contact information.

Reply:
updated in [commit](https://github.com/MyoHub/myosuite/commit/e31145ea1a059add2cb32370ebb6951a8b3071a7)